Networking/Networking/Remote/CustomerRemote.swift

```diff
@@ -4,6 +4,31 @@ import Foundation
 ///
 public class CustomerRemote: Remote {
 
+    /// Create a customer.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll create a customer.
+    ///     - customer: The Customer model used to create the custom entity for the request.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func createCustomer(for siteID: Int64,
```

Comment:
It would be better to follow the same order declared everywhere else, declaring first the `getAllCustomers` method, and after this one the `createCustomer` method.

Reply:
Updated order in 5c8f176.